products/compute/api.yaml

```diff
@@ -3488,6 +3488,86 @@ objects:
         description: 'The write-only private key in PEM format.'
         required: true
         input: true
+  - !ruby/object:Api::Resource
+    # This is intentionally out of alphabetic order.
+    name: 'ManagedSslCertificate'
+    kind: 'compute#sslCertificate'
+    min_version: beta
+    base_url: projects/{{project}}/global/sslCertificates
+    collection_url_response: !ruby/object:Api::Resource::ResponseList
+      kind: 'compute#sslCertificateList'
+      items: 'items'
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/load-balancing/docs/ssl-certificates'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/v1/sslCertificates'
+    input: true
+    has_self_link: true
+    description: |
+      An SslCertificate resource, used for HTTPS load balancing.  This resource
+      represents a certificate for which the certificate secrets are created and
+      managed by Google.
+<%= indent(compile_file({timeouts: {
+                        insert_sec: 6 * 60,
+                        update_sec: 6 * 60,
+                        # Deletes can take up to 30 minutes to complete, since they depend
+                        # on the provisioning process either succeeding or failing completely.
+                        delete_sec: 30 * 60}}, 'templates/global_async.yaml.erb'), 4) %>
```

Comment:
Should we use 45 minutes to give a little more buffer?

Reply:
It should be ~20 or so, empirically - 30 is decently buffered.